src/data-table/data-table.tsx

```diff
@@ -28,27 +32,55 @@ export const DataTable = ({
     loadState = "success",
     enableMultiSelect,
     selectedIds,
+    disabledIds,
     enableSelectAll,
+    enableSelectionBar,
     emptyView,
+    selectionBarContent,
+    enableStickyHeader = false,
     renderCustomEmptyView,
     onHeaderClick,
     onSelect,
     onSelectAll,
+    onClearSelectionClick,
     ...otherProps
 }: DataTableProps) => {
+    // =============================================================================
+    // CONST, STATE, REF
+    // =============================================================================
+    const [isAtBottom, setIsAtBottom] = useState(false);
+    const [showLastBorder, setShowLastBorder] = useState(false);
+    const tableRef = useRef<HTMLInputElement>();
+
+    // =============================================================================
+    // EFFECTS
+    // =============================================================================
+    useEffect(() => {
+        scrollHandler();
+        checkLastBorder();
```

Comment:
what does this do? also if row height changes, does this need to be recomputed?

Reply:
Yes. I believe, if rows get changed. This need to be  re computed